### DISCUSSION TITLE
Old Windows Download

### DISCUSSION THREAD

**Discussion by johan-lejdung** [lllyasviel/Fooocus#3108] [Q&A] [answered]
The windows download in the readme seems to be stuck on version 2.1.831. How do I download newer versions?

If I modify the download link to be https://github.com/lllyasviel/Fooocus/releases/download/release/Fooocus_win64_v2-4-2.7z it doesnt find anything to download.

**Comment by mashb1t** [comment, marked as answer]
The link https://github.com/lllyasviel/Fooocus/releases/download/release/Fooocus_win64_2-1-831.7z is correct, Fooocus will auto-update on startup.

**Reply by johan-lejdung** [reply]
Oh, okay! Anyway I can see the version that is running?

**Reply by mashb1t** [reply]
sure, you can find an output in the console log on startup as well as in the file `fooocus_version.txt` and in the browser tab title.
